packages/module/src/components/nodes/labels/NodeLabel.tsx

```diff
@@ -10,8 +10,10 @@ import LabelContextMenu from './LabelContextMenu';
 import LabelIcon from './LabelIcon';
 import LabelActionIcon from './LabelActionIcon';
 import { BadgeLocation, LabelPosition, NodeStatus } from '../../../types';
+import { Node } from '../../../types';
```

Comment:
could this be combined with the line above it?

Reply:
Yes, nice catch @nicolethoen ! Updated